### PR TITLE
[CL Incentives] Add 0 second to supported uptime

### PR DESCRIPTION
**Pull request by AlpinYukseloglu**
Closes: #4640

## What is the purpose of the change

This PR adds a 0 sec (1 nanosecond) supported uptime for CL incentives.

## Brief Changelog

- Add 1 nanosecond as a supported uptime

## Testing and Verifying

- The vast majority of tests were compatible with changing supported uptimes, so the change is already covered by them!

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? (no)
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (no)
  - How is the feature or change documented? (not documented)

### CODE REVIEW

File: x/concentrated-liquidity/incentives_test.go [hunk 1]
```diff
@@ -1753,6 +1753,12 @@ func (s *KeeperTestSuite) TestInitOrUpdatePositionUptime() {
 					// 100 + 103 + UGI = 303
 					sdk.NewDecCoin("foo", sdk.NewInt(303)),
 				),
+				sdk.NewDecCoins(
+					// 100 + 112 + UGI = 312
+					sdk.NewDecCoin("bar", sdk.NewInt(309)),
+					// 100 + 104 + UGI = 304
+					sdk.NewDecCoin("foo", sdk.NewInt(303)),
```
Comment (p0mvn) [inline x/concentrated-liquidity/incentives_test.go:1760]: The values do not match the comment

**Review by p0mvn — APPROVED** [review]

File: x/concentrated-liquidity/incentives_test.go (hunk 1 above)
Comment (AlpinYukseloglu) [reply x/concentrated-liquidity/incentives_test.go]: Ah nice catch – it turns out it doesn't matter for this test case, but I triple-checked that the test performs as expected. Also fixed in most recent commit.

**Review by czarcas7ic — APPROVED** [review]

File: x/concentrated-liquidity/types/constants.go [hunk 2]
```diff
@@ -33,7 +33,7 @@ var (
 	MaxSpotPrice              = sdk.MustNewDecFromStr("100000000000000000000000000000000000000")
 	MinSpotPrice              = sdk.MustNewDecFromStr("0.000000000000000001")
 	// Supported uptimes preset to 1 min, 1 hr, 1D, 1W
```
[the remark this answers is not in the document]
Comment (stackman27) [reply x/concentrated-liquidity/types/constants.go:35]: isn't it nanosecond so `1ns1?

[the remark this answers is not in the document]
Comment (czarcas7ic) [reply x/concentrated-liquidity/types/constants.go:35]: modified, thanks

**Pull request merged**